helm-chart/kuberay-operator/templates/multiple_namespaces_role.yaml

```diff
@@ -0,0 +1,240 @@
+# Install Role for namespaces listed in watchNamespace.
+# This should be consistent with `role.yaml`, except for the `kind` field.
+{{- if and .Values.rbacEnable .Values.singleNamespaceInstall }}
+{{- $watchNamespaces := default (list .Release.Namespace) .Values.watchNamespace }}
+{{- range $namespace := $watchNamespaces }}
+---
+kind: Role
```

Comment:
I'm wondering if we could add thescenario: `singleNamespaceInstall is set to false or not set` here. So that there is no need to synchronize `multiple_namespaces_role.yaml` and  `role.yaml` (same as `rolebinding`)

More specifically, when `singleNamespaceInstall` is false:

- set $watchNamespaces to a placeholder value (so that only iterates once) 
- create a ClusterRole instead of a Role. 
- omit the metadata.namespace filed

Reply:
I originally want to do this by updating `role.yaml` and `rolebinding.yaml`. However, it can be extremely challenging or even impossible (?) to achieve that. The Helm parser is not smart enough.